docs/installation/installation-using-docker-macos.md

```diff
@@ -27,42 +27,56 @@ There are two domains each for different language in default installation. First
 sudo ifconfig lo0 alias 127.0.0.2 up
 ```
 
-### 2. Create docker-compose.yml file
+### 2. Create docker-compose.yml and docker-sync.yml
 Create `docker-compose.yml` from template [`docker-compose-mac.yml.dist`](../../project-base/docker/conf/docker-compose-mac.yml.dist).
 ```
 cp docker/conf/docker-compose-mac.yml.dist docker-compose.yml
 ```
 
-### 3. Compose Docker container
-On MacOS you need to synchronize folders using docker-sync.
-Before starting synchronization you need to create a directory for persisting Postgres data so you won't lose it when the container is shut down.
-```
-mkdir -p var/postgres-data
-docker-sync start
-```
+Create `docker-sync.yml` from template [`docker-sync.yml.dist`](../../project-base/docker/conf/docker-sync.yml.dist).
+
+### 2.1 Set the UID and GID to allow file access in mounted volumes
+Because we want both the user in host machine (you) and the user running php-fpm in the container to access shared files, we need to make sure that they both have the same UID and GID.
+This can be achieved by build arguments `www_data_uid` and `www_data_gid` that should be set to the UID and GID as your own user in your `docker-compose.yml`.
+Also, you need to change `sync_userid` in `docker-sync.yml` file.
+
+You can find out your UID by running `id -u` and your GID by running `id -g`.
+
+#### 2.1 Fill your credentials into files
+You need to insert your user id and group id into these two newly created file.
 
-Then start containers
+Get user id:
 ```
-docker-compose up -d
+id -u
 ```
 
-### 4. Set file permissions
-Grant system users inside the container the required permissions
-#### Connect into terminal of the Docker container
+Get group id:
 ```
-docker exec -it shopsys-framework-php-fpm bash
+id -g
 ```
 
-#### Allow user with UID 33 ("www-data" in "php-fpm" container) read and write all project files
+Set these values into your `docker-compose.yml` into `php-fpm` container definition and replace this values in `args` section.
+
+Also you need to insert your user id into `docker-sync.yml` into value `sync_userid`.
+
+### 3. Set the UID and GID to allow file access in mounted volumes
```

Comment:
Since we've opened up the topic of duplicates, this is a copy of the upper section 2.1 without the mention of the `docker-sync.yml`... This needs some fixing up.